Quadriga/src/main/java/edu/asu/spring/quadriga/rest/DictionaryRestController.java

```diff
@@ -193,7 +212,7 @@
      * @param dictionaryId
      * @param model
      * @return
-     * @throws RestException
+     * @throws Exception

```

Comment:
This was a mistake from our side due to lot of merge conflicts. Let me know if we still miss anything.